lib/src/main/java/com/adyen/nexoapp/lib/model/transactions/RefundToACardModel.kt

```diff
@@ -0,0 +1,48 @@
+package com.adyen.nexoapp.lib.model.transactions
+
+import android.app.Application
+import android.content.Context
+import com.adyen.nexoapp.lib.model.api.SaleToPoiRequest
+import com.adyen.nexoapp.lib.model.api.TerminalRequest
+import com.adyen.nexoapp.lib.model.api.body.payment.*
+import com.adyen.nexoapp.lib.model.api.header.MessageCategory
+import com.adyen.nexoapp.lib.model.api.header.MessageHeader
+import com.adyen.nexoapp.lib.util.Random
+import com.adyen.nexoapp.lib.util.toUtcTimeStamp
+import java.util.*
+
+class RefundToACardModel(val application: Application,
+                   val poiId: String,
+                   val currency: String,
+                   val amount: Double) : TransactionModel {
+
+    private val saleId by lazy {
```

Comment:
Maybe you can refactor this and move it into `TransactionModel`?

Reply:
You mean the saleId ? I will have a look.